packages/store-indexer/src/koa-middleware/metrics.ts

```diff
@@ -61,6 +63,16 @@ export function metrics({
     blockTagGauge.set(blockTagToValue[followBlockTag]);
   }
 
+  if (getDistanceFromFollowBlock != null) {
+    new promClient.Gauge({
+      name: "distance_from_follow_block",
+      help: "Block distance from the block tag this the indexer is following",
+      async collect(): Promise<void> {
+        this.set(Number(await getDistanceFromFollowBlock()));
```

Comment:
does `this.set` only support number? vs e.g. `BigInt.toString()`

Reply:
I think number is better here so we can do things like `>` for the alerts (and very unlikely the numbers ever get into a range where they don't fit into a number anymore)

Reply:
oh yes, I just assumed that what "gauge" metric type is for, not the value we pass into the gauge